settings.php

```diff
@@ -38,54 +40,30 @@
     // Create a category in the admin tree
     $paymentplugincat = new admin_category('tool_paymentplugin_folder', get_string('pluginname', 'tool_paymentplugin'), false);
     $paymentplugincat->add('tool_paymentplugin_folder', $globalsettings);
+
     // Add the category to the tree
     $ADMIN->add('tools', $paymentplugincat);
 
-
-    // Create settings
-    $heading = new admin_setting_heading('tool_paymentplugin_gsettings/heading', $globalsettings->visiblename,
-        get_string('gsettingsdesc', 'tool_paymentplugin'));
-
-    $disableallcheck = new admin_setting_configcheckbox('tool_paymentplugin_gsettings/disablePurchases', get_string('gsettingsdisablepurchase', 'tool_paymentplugin'),
-        get_string('gsettingsdisablepurchasedesc', 'tool_paymentplugin'), 0);
-
-    $checkbox2 = new admin_setting_configcheckbox('tool_paymentplugin_gsettings/checkbox2', get_string('gsettingscheck2', 'tool_paymentplugin'),
-        get_string('gsettingscheck2desc', 'tool_paymentplugin'), 0);
-
-    $exampleselections = [
-            'Option A' => get_string('gsettingsmulti1selectionA', 'tool_paymentplugin'),
-            'Option B' => get_string('gsettingsmulti1selectionB', 'tool_paymentplugin'),
-            'Option C' => get_string('gsettingsmulti1selectionC', 'tool_paymentplugin')
-        ];
-    $multiselect = new admin_setting_configmultiselect('tool_paymentplugin_gsettings/multi1', get_string('gsettingsmulti1', 'tool_paymentplugin'),
-        get_string('gsettingsmulti1desc', 'tool_paymentplugin'), [], $exampleselections);
-
-    $entryfield = new admin_setting_configtext('tool_paymentplugin_test_entryfield', 'Entry Field:', 'The box below will hold this value', '');
-    if ($entryfield->get_setting() == '') {
-        $entryfield->write_setting("For Example...");
+    // Sub Plugin Enabled/Disabled Settings
+    // Create Configs
+    $gateways = paymentgateway::get_all_gateway_objects();
+
+    $globalsettings->add(new admin_setting_heading('tool_paymentplugin_subsettings/heading', get_string('tool_paymentplugin_subsettings/heading', 'tool_paymentplugin'),
+        count($gateways).get_string('tool_paymentplugin_subsettings/headingdesc', 'tool_paymentplugin').' '.
+        count(paymentgateway::get_all_enabled_gateway_objects()).' '.get_string('tool_paymentplugin_subsettings/headingdesc2', 'tool_paymentplugin')));
```

Comment:
This should probably be different when "Disable All Payment Gateways" is enabled, otherwise it says that there's payment gateways enabled when there aren't.